storage/aptosdb/src/state_store/mod.rs

```diff
@@ -63,6 +63,69 @@ impl StateStore {
         ))
     }
 
+    /// Get the state value given the state key and root hash of state Merkle tree by using the
+    /// state value index. Only used for testing for now but should replace the
+    /// `get_value_with_proof_by_version` call for VM execution to fetch the value without proof.
+    #[cfg(test)]
+    pub fn get_value_by_version(
+        &self,
+        state_key: &StateKey,
+        version: Version,
+    ) -> Result<Option<StateValue>> {
+        match self.get_jmt_leaf_node_key(state_key, version)? {
+            Some(node_key) => {
+                let mut iter = self
+                    .db
+                    .iter::<JellyfishMerkleNodeSchema>(Default::default())?;
+                iter.seek_for_prev(&node_key)?;
```

Comment:
Should just `get()` and expect it to be there, right?

Reply:
Haha - my bad. Thanks for pointing it out.